src/main/java/no/digipost/api/client/representations/accounts/UserAccount.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (C) Posten Norge AS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package no.digipost.api.client.representations.accounts;
+
+import no.digipost.api.client.representations.DigipostAddress;
+import no.digipost.api.client.representations.EncryptionKey;
+
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement(name = "user-account")
+public class UserAccount {
+
+	@XmlElement(name = "digipost-address")
+	private final DigipostAddress digipostAddress;
+    @XmlElement(name = "encryption-key")
```

Comment:
File mixes tabs/spaces.
The .editorconfig says `indent_style = space`, but it seems to be mostly tabs in a lot of new files.